src/app/templates/SwapForm/SwapFormInput/AssetsMenu/AssetsMenu.tsx

```diff
@@ -85,19 +85,32 @@ export const AssetsMenu: FC<Props> = ({
             <p className="flex items-center justify-center text-gray-600 text-base font-light">
               {searchString ? <SearchIcon className="w-5 h-auto mr-1 stroke-current" /> : null}
 
-              <span>{showTokenIdInput ? <T id="specifyTokenId" /> : <T id="noAssetsFound" />}</span>
+              <span>
+                <T id={showTokenIdInput ? 'specifyTokenId' : 'noAssetsFound'} />
+              </span>
             </p>
           )}
         </div>
       )}
+
       <List
         width={dropdownWidth}
         height={240}
         rowCount={options.length}
-        rowHeight={65}
-        rowRenderer={({ key, index, style }) => (
-          <AssetOption key={key} assetSlug={options[index]} style={style} onClick={handleOptionClick} />
-        )}
+        rowHeight={64}
+        rowRenderer={({ key, index, style }) => {
+          const option = options[index];
+
+          return (
+            <AssetOption
+              key={key}
+              assetSlug={option}
+              selected={value === option}
+              style={style}
+              onClick={handleOptionClick}
+            />
+          );
+        }}
```

Comment:
Can we create a component for this logic or move this logic into separate function?